### PR TITLE
Add class names for ui tests

### PR DESCRIPTION
**Pull request by dhivya2016**
Added new class names for done and delete button.

### CODE REVIEW

File: swagger-api-tests/src/test/generated-api.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
+"use strict";
```
Comment (ikhripunov) [inline swagger-api-tests/src/test/generated-api.js]: Add to .gitignore, as this should be generated every time

File: swagger-api-tests/src/test/generated-api.js.map [hunk 2]
```diff
@@ -0,0 +1 @@
...AAK,SAAS,CAAC,CAAC,CAAC;gBACnC,IAAI,GAAG,UAAU,CAAC,MAAM,CAAC,CAAC;YAC9B,CAAC;YAED,EAAE,CAAC,CAAC,UAAU,CAAC,gBAAgB,CAAC,CAAC,CAAC;gBAC9B,MAAM,CAAC,IAAI,CAAC,UAAU,CAAC,gBAAgB,CAAC,CAAC,OAAO,CAAC,UAAS,aAAa;oBACnE,eAAe,CAAC,aAAa,CAAC,GAAG,UAAU,CAAC,gBAAgB,CAAC,aAAa,CAAC,CAAC;gBAChF,CAAC,CAAC,CAAC;YACP,CAAC;YAED,IAAI,GAAG,eAAe,CAAC;YACvB,eAAe,GAAG,EAAE,CAAC;YAErB,IAAI,CAAC,OAAO,CAAC,MAAM,EAAE,MAAM,GAAG,IAAI,EAAE,IAAI,EAAE,OAAO,EAAE,eAAe,EAAE,IAAI,EAAE,MAAM,EAAE,OAAO,CAAC,CAAC;QAC/F,CAAC,CAAC,CAAC;IACP,CAAC;IAED,QAAQ,CAAC,UAGR;QACG,MAAM,MAAM,GAAG,UAAU,CAAC,OAAO,GAAG,UAAU,CAAC,OAAO,GAAG,IAAI,CAAC,MAAM,CAAC;QACrE,IAAI,IAAI,GAAG,YAAY,CAAC;QACxB,IAAI,IAAS,CAAC;QACd,IAAI,eAAe,GAAQ,EAAE,CAAC;QAC9B,IAAI,OAAO,GAAQ,EAAE,CAAC;QACtB,IAAI,IAAI,GAAQ,EAAE,CAAC;QACnB,MAAM,CAAC,IAAI,OAAO,CAAC,CAAC,OAAO,EAAE,MAAM;YAE/B,EAAE,CAAC,CAAC,UAAU,CAAC,gBAAgB,CAAC,CAAC,CAAC;gBAC9B,MAAM,CAAC,IAAI,CAAC,UAAU,CAAC,gBAAgB,CAAC,CAAC,OAAO,CAAC,UAAS,aAAa;oBACnE,eAAe,CAAC,aAAa,CAAC,GAAG,UAAU,CAAC,gBAAgB,CAAC,aAAa,CAAC,CAAC;gBAChF,CAAC,CAAC,CAAC;YACP,CAAC;YAED,IAAI,CAAC,OAAO,CAAC,KAAK,EAAE,MAAM,GAAG,IAAI,EAAE,IAAI,EAAE,OAAO,EAAE,eAAe,EAAE,IAAI,EAAE,MAAM,EAAE,OAAO,CAAC,CAAC;QAC9F,CAAC,CAAC,CAAC;IACP,CAAC;CAEJ;AAtJD,0BAsJC"}
```
Comment (ikhripunov) [inline swagger-api-tests/src/test/generated-api.js.map]: Add to .gitignore

File: swagger-api-tests/swagger.json [hunk 3]
```diff
@@ -0,0 +1,122 @@
+{
```
Comment (ikhripunov) [inline swagger-api-tests/swagger.json]: I thought the idea is to use 1 swagger file/artifact for all the modules. There's swagger.json in root folder already

File: swagger-api-tests/yarn.lock [hunk 4]
```diff
@@ -0,0 +1,1598 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (ikhripunov) [inline swagger-api-tests/yarn.lock]: I'm pretty sure all *.lock files could be added to .gitignore

File: swagger-api-tests/package-lock.json [hunk 5]
```diff
@@ -0,0 +1,1499 @@
+{
```
Comment (ikhripunov) [inline swagger-api-tests/package-lock.json]: I'm pretty sure all *.lock files could be added to .gitignore

File: swagger-ui-tests/package-lock.json [hunk 6]
```diff
@@ -0,0 +1,1377 @@
+{
```
Comment (ikhripunov) [inline swagger-ui-tests/package-lock.json]: I'm pretty sure all *.lock files could be added to .gitignore

File: swagger-api-tests/package-lock.json (hunk 5 above)
Comment (dhivya2016) [reply swagger-api-tests/package-lock.json]: Added all the lock files to .gitignore

Comment (davidpkatz) [reply swagger-api-tests/package-lock.json]: I asked @rvowles about this and he said it was a good idea to include them as they lock the dependency tree. See what he says though

Comment (davidpkatz) [reply swagger-api-tests/package-lock.json]: Please do commit the .lock files, don't add them to .gitignore. This ensures builds are repeatable

Comment (dhivyach) [reply swagger-api-tests/package-lock.json]: Updated this to include lock files in 'ui-tests-po' branch

File: swagger-api-tests/src/test/generated-api.js.map [hunk 7]
```diff
@@ -1 +1 @@
...
...,IAAI,GAAG,UAAU,CAAC,MAAM,CAAC,CAAC;YAC9B,CAAC;YAED,EAAE,CAAC,CAAC,UAAU,CAAC,gBAAgB,CAAC,CAAC,CAAC;gBAC9B,MAAM,CAAC,IAAI,CAAC,UAAU,CAAC,gBAAgB,CAAC,CAAC,OAAO,CAAC,UAAS,aAAa;oBACnE,eAAe,CAAC,aAAa,CAAC,GAAG,UAAU,CAAC,gBAAgB,CAAC,aAAa,CAAC,CAAC;gBAChF,CAAC,CAAC,CAAC;YACP,CAAC;YAED,IAAI,GAAG,eAAe,CAAC;YACvB,eAAe,GAAG,EAAE,CAAC;YAErB,IAAI,CAAC,OAAO,CAAC,MAAM,EAAE,MAAM,GAAG,IAAI,EAAE,IAAI,EAAE,OAAO,EAAE,eAAe,EAAE,IAAI,EAAE,MAAM,EAAE,OAAO,CAAC,CAAC;QAC/F,CAAC,CAAC,CAAC;IACP,CAAC;IAED,QAAQ,CAAC,UAGR;QACG,MAAM,MAAM,GAAG,UAAU,CAAC,OAAO,CAAC,CAAC,CAAC,UAAU,CAAC,OAAO,CAAC,CAAC,CAAC,IAAI,CAAC,MAAM,CAAC;QACrE,IAAI,IAAI,GAAG,YAAY,CAAC;QACxB,IAAI,IAAS,CAAC;QACd,IAAI,eAAe,GAAQ,EAAE,CAAC;QAC9B,IAAI,OAAO,GAAQ,EAAE,CAAC;QACtB,IAAI,IAAI,GAAQ,EAAE,CAAC;QACnB,MAAM,CAAC,IAAI,OAAO,CAAC,CAAC,OAAO,EAAE,MAAM,EAAE,EAAE;YAEnC,EAAE,CAAC,CAAC,UAAU,CAAC,gBAAgB,CAAC,CAAC,CAAC;gBAC9B,MAAM,CAAC,IAAI,CAAC,UAAU,CAAC,gBAAgB,CAAC,CAAC,OAAO,CAAC,UAAS,aAAa;oBACnE,eAAe,CAAC,aAAa,CAAC,GAAG,UAAU,CAAC,gBAAgB,CAAC,aAAa,CAAC,CAAC;gBAChF,CAAC,CAAC,CAAC;YACP,CAAC;YAED,IAAI,CAAC,OAAO,CAAC,KAAK,EAAE,MAAM,GAAG,IAAI,EAAE,IAAI,EAAE,OAAO,EAAE,eAAe,EAAE,IAAI,EAAE,MAAM,EAAE,OAAO,CAAC,CAAC;QAC9F,CAAC,CAAC,CAAC;IACP,CAAC;CAEJ;AAtJD,0BAsJC"}
```
Comment (ikhripunov) [inline swagger-api-tests/src/test/generated-api.js.map]: This should be removed from tracked files

**Pull request closed without merging**